examples/MDX/test-mdx.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import slides from './slides.mdx';
-import Deck from '../../src/components/Deck';
-import Slide from '../../src/components/Slide';
+import Deck from '../../src/components/deck';
```

Comment:
This is still `/MDX/*` which doesn't solve the problem.

Reply:
It's lower case locally. I tried to make it upper then lower case again. No diff in git.

<img width="278" alt="Screen Shot 2019-08-12 at 3 22 34 PM" src="https://user-images.githubusercontent.com/6595948/62899468-107b7500-bd15-11e9-870b-16d47a8cbe86.png">

Reply:
Fixed!